addons/udes_stock/README.md

```diff
@@ -115,6 +115,7 @@ A lot of custom UDES functionality is specfied at the picking type level. This i
 | u_validate_real_time       | boolean | Do we validate move lines in real time. |
 | u_drop_location_constraint | string  | Whether drop location should be scanned, suggested and, then, enforced (enum: 'dont_scan', 'scan', 'enforce', 'suggest'); default: 'scan'. |
 | u_drop_location_policy     | string  | To indicate the policy for suggesting drop locations (enum: 'exactly_match_move_line', 'by_products', 'by_packages'); default: 'exactly_match_move_line'. |
+|u_preprocess_drop_location  | boolean | Selects if suggestions u_drop_location_policy should be added on assignment |
```

Comment:
Bikeshedding: I would call this `u_drop_lcoation_preprocess` so it starts like the 2 entries above. It would possibly help searching and make it clear that the 3 things are related.
No strong opinion though.

Reply:
I would add a few details about how you add the suggestions (i.e. use the first suggestion as the `location_dest_id` of the move lines).

Reply:
Also, bad alignment (missing space before field name). I'll stop here with the style comments, promised.